.circleci/config.yml

```diff
@@ -130,6 +130,28 @@ jobs:
             -H 'Accept: application/json' \
             https://rahti.csc.fi:8443/api/v1/namespaces/rems-dev/pods/${DEV_CONTAINER_NAME}
 
+  rahti-demo:
+    <<: *defaults
+    steps:
+      - checkout
+      - attach_workspace:
+          at: .
+      - run: docker build --pull --tag docker-registry.rahti.csc.fi/rems/rems:release --tag docker-registry.rahti.csc.fi/rems/rems:$(git describe --abbrev=0 --tags) .
```

Comment:
do we really need to rebuild the image here? can't we just retag the existing one? perhaps we could have three jobs:
- rahti: builds image, pushes it to rahti, tags with various tags (e.g. release if we are on a `v9.9` tag)
- rahti-dev: updates remsdev to run latest `circle` tag
- rahti-demo: updates rems-demo to run lates `release` tag

Reply:
I get what you mean and my logic here was off. While we could separate the build to it's own job it would complicate the process somewhat. How do we know if circle was triggered my a build or a tag? I tried to modify the logic so that only one of either rahti-dev or rahti-demo would be run. This does achieve the same end result, but arguably is not as "neat".

Reply:
you can check if a tag points to master using e.g. `git describe --exact-match`

Reply:
relatedly, the tagging & pushing could be in its own script under `.circle` for easier development

Reply:
Good idea. I moved the commands to an other file.